alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -220,8 +221,8 @@ pub fn new(config: &PtyConfig, window_size: WindowSize) -> Option<Pty> {
         }
     }
 
-    let conin = EventedAnonWrite::new(conin);
-    let conout = EventedAnonRead::new(conout);
+    let conin = UnblockedWriter::new(conin, PIPE_CAPACITY);
+    let conout = UnblockedReader::new(conout, PIPE_CAPACITY);
```

Comment:
Are we double-buffering the PTY on Windows? That seems pretty inefficient.

Have you done any performance testing on Windows?

Reply:
That's sort of true, but it was the case before, nothing changed here.